src/xrGame/map_manager.h

```diff
@@ -16,7 +16,7 @@ class CMapManager
     CMapManager();
     ~CMapManager();
     void __stdcall Update();
-    /*ICF */ Locations& Locations(); //{return *m_locations;}
+    /*ICF */ Locations& getLocations(); //{return *m_locations;}
```

Comment:
What's the problem with this functions?

Anyway, it's better to name it with PascalCase: `GetLocations` (according to the code style)

Reply:
Problem with same name of returning type and method. Ok, PascalCase acceptable.